src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -122,6 +159,45 @@ contract SecurityCouncilManager is
         for (uint256 i = 0; i < _securityCouncils.length; i++) {
             _addSecurityCouncil(_securityCouncils[i]);
         }
+
+        _setMinRotationPeriod(_minRotationPeriod);
+
+        // we do our own 712 functionality because inheriting the OZ version
+        // would change our storage layout
+        NAME_HASH = keccak256(bytes("SecurityCouncilManager"));
+        VERSION_HASH = keccak256(bytes("1"));
```

Comment:
these can simply be constants

Reply:
not critical, but if we change these to constant we should remember to update `__gap` too

Reply:
I have them as storage so that we do the same as OZ's library. Dunno if we'll ever need to change the name or version, but I guess that's why OZ do that. What dyou guys think, should I make constant?

Reply:
OZ did that because it is an upgradable contract that need to be initialized. In our case we already know what to be initialized into.

Reply:
updated